seed/static/seed/js/controllers/inventory_detail_controller.js

```diff
@@ -500,16 +500,32 @@ angular.module('BE.seed.controller.inventory_detail', [])
       };
 
```

Comment:
General comment,
Here's the diff I got from importing and exporting the same file - I don't know enough about the BS schema to know if this is expected, so feel free to "resolve" this comment if you think it looks good:
<img width="1440" alt="Screen Shot 2020-05-16 at 9 18 02 PM" src="https://user-images.githubusercontent.com/30608004/82241755-eedd9880-98f9-11ea-8ccc-b3f4e178d8c6.png">